lab-justin/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "lab-justin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "greet.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "gulp": "^3.9.1",
+    "gulp-mocha": "^2.2.0",
+    "gulp-watch": "^4.3.5"
```

Comment:
You aren't using the gulp-watch package, you're using the watch method built into gulp. This is something different.